bin/embedding-sdk/fixup-types-after-compilation.js

```diff
@@ -14,14 +14,12 @@ const SDK_PACKAGE_NAME = "@metabase/embedding-sdk-react";
 
 // this map should be synced with "tsconfig.sdk.json"
 const REPLACES_MAP = {
-  "metabase-enterprise/": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/metabase-enterprise/`,
-  "metabase-lib/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-lib/`,
+  "metabase-enterprise": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/metabase-enterprise`,
   "metabase-lib": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-lib`,
-  "metabase-shared/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-shared/`,
-  "metabase-types/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-types/`,
-  "metabase/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase/`,
-  "embedding-sdk/": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/embedding-sdk/`,
-  "cljs/": `${SDK_PACKAGE_NAME}/dist/target/cljs_release/`,
+  "metabase-types": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-types`,
+  "metabase": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase`,
+  "embedding-sdk": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/embedding-sdk`,
+  "cljs": `${SDK_PACKAGE_NAME}/dist/target/cljs_release`,
```

Comment:
Removed the slash here so we can only have a single variant. We'll append it below.